.github/workflows/run-tests.yml

```
@@ -47,6 +47,11 @@ jobs:
       - name: Run acceptance tests
         run: |
           cargo test --features contract_test --test contract_test -- -t "not @na=rust and @featureSet=publish"
+      - name: Expose acceptance tests reports
+        uses: actions/upload-artifact@v3
+        with:
+          name: acceptance-test-reports
+          path: tests/report.xml
```

Reports stored in `tests` better to have some different names, which will make it possible to identify whether it is from regular or `beta` tests.

```suggestion
          path: tests/report.xml
          retention-days: 7
```

---

Was thinking of making beta test result report as tests/report-beta.xml
Does this make sense ?
Or We can make report-required.xml ?

---

`tests/report.xml` and `tests/report-beta.xml` are good names ;)